rosetta-source/src/main/rosetta/product-template-func.rosetta

```diff
@@ -114,3 +115,15 @@ func FpmlIrd8: <"FpML validation rule ird-8 - If the same party is specified as
         accounts Account (0..*)
     output:
         success boolean (1..1) <"Validation result">
+
+func ExpirationTimeType:
+    inputs: 
+        expirationTime BusinessCenterTime (0..1)
+        expirationTimeType ExpirationTimeTypeEnum (0..1)
+    output: 
+        opCheck boolean (1..1)
+    
+    set opCheck:
```

Comment:
A plain-english description of the logic would be welcome. This is checking the correlation between the expiration time type being 'specific' and the existence of an expiration time.